android/app/src/main/java/com/example/clarity/sdk/Utils.kt

```diff
@@ -1,7 +1,8 @@
 package com.example.clarity.sdk
 
 data class SetMetadata(val set_id: Int, val title: String, val type: String, val is_public: Boolean, val likes: Int)
-data class UserWithId(val user_id: Int, val username: String, val email: String, val firstname: String, val lastname: String, val phone_number: String, val login_streak: Int)
+data class User(val username: String, val email: String, val password: String, val firstname: String, val lastname: String, val phone_number: String, val difficulty: Difficulty)
```

Comment:
is there a specific reason as to why we would use User object instead of UserWithId?

Reply:
User object is used when creating a new user afaik, UserWithId is more of an object that represents a user in the db. Good question though, this is quite confusing so lemme j make sure where the User object is used and i'll make changes if required

Reply:
fixed